NSString+MD5.h

```diff
@@ -7,8 +7,25 @@
 
 @import Foundation;
 
+
+#ifdef NS_ASSUME_NONNULL_BEGIN
+#define MD5_DIGEST_ASSUME_NONNULL_BEGIN NS_ASSUME_NONNULL_BEGIN
+#else
+#define MD5_DIGEST_ASSUME_NONNULL_BEGIN
+#endif
+
+#ifdef NS_ASSUME_NONNULL_END
+#define MD5_DIGEST_ASSUME_NONNULL_END NS_ASSUME_NONNULL_END
+#else
+#define MD5_DIGEST_ASSUME_NONNULL_END
+#endif
+
+MD5_DIGEST_ASSUME_NONNULL_BEGIN
+
 @interface NSString (MD5)
 
 - (NSString *)MD5Digest;
 
 @end
+
+MD5_DIGEST_ASSUME_NONNULL_END
```

Comment:
I _think_ that forgetting to add `NS_ASSUME_NONNULL_END` would result in an error. I’ll check when I change the first section.